Makefile

```diff
@@ -45,6 +45,7 @@ BUILD_IMAGE ?= golang:1.11.2
 HYPERFED_TARGET = bin/hyperfed
 CONTROLLER_TARGET = bin/controller-manager
 KUBEFEDCTL_TARGET = bin/kubefedctl
+WEBHOOK_TARGET = bin/webhook
```

Comment:
(No action required) I think unique binary names would be preferable (e.g. `kubefed-controller-manager`, `kubefed-webhook`).

Reply:
I think this should eventually be `admission-webhook`. I was going to do this in this PR, but then decided against it because it would require renaming several files and their import paths. The main thing is that the user visible name is `admission-webhook`.